### DISCUSSION TITLE
Harpoon guns on back (Toolbelt slot)

### DISCUSSION THREAD

**Discussion by Fabian272** [FakeFishGames/Barotrauma#8318] [Ideas and suggestions]
I've seen people running around with the Autoshotgun, Fissile Accelerator, and Grenade Launcher on their back and thought it would be neat if you could put the harpoon guns and and maybe few other ones on your back too like that.

**Comment by Abkim27** [comment]
I think it would be cool with all weapons, so you need to think, take some melee weapons and toolbelt, or take a ranged weapon without toolbelt.
